### PR TITLE
added: Contributors  section on  the challenge details page#71

### PR DESCRIPTION
**Pull request by Bivas-Biswas**
issue #71
Task completed:
- [x] create separate component of ContributorProfile
- [x]  responsive
- [x] dynamic data

### CODE REVIEW

**Comment by Bivas-Biswas** [comment]
what happen @rishipurwar1

File: src/components/smallComponents/ContributorProfile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Image, Placeholder, Transformation } from "cloudinary-react"
+
+function ContributorProfile(props) {
```
Comment (rishipurwar1) [inline src/components/smallComponents/ContributorProfile.js:4]: Please use arrow function and directly destructure contributors in the argument `function ContributorProfile({contributors}) {` and then use it like this `const { name, profilePhoto, title } = contributors`
Nothing wrong with your code, just to maintain the coding style

**Review by rishipurwar1 — COMMENTED** [review]
> what happened @rishipurwar1

nothing

**Comment by Bivas-Biswas** [comment]
now check

**Comment by rishipurwar1** [comment]
> now check

looks good to me, going to merge

**Comment by rishipurwar1** [comment]
Hey @Bivas-Biswas 

I merged the changes but I think your last commit got merged.

**Comment by rishipurwar1** [comment]
Is it possible to reopen the pr?

**Comment by rishipurwar1** [comment]
wait! No need to do anything from your side.
I didn't add to the production firebase db.

**Comment by rishipurwar1** [comment]
@Bivas-Biswas I just test the changes in the development mode, everything looks good to me but you just need to add a social link to the name and add an underline to the name, if the underline looks good then keep otherwise add underline on hover.

**Comment by Bivas-Biswas** [comment]
where to add hyperlink whole component or only name

**Comment by rishipurwar1** [comment]
only name

**Pull request merged**